src/qt/pivx/navmenuwidget.cpp

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) 2019 The PIVX developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include "qt/pivx/navmenuwidget.h"
+#include "qt/pivx/forms/ui_navmenuwidget.h"
+#include "qt/pivx/PIVXGUI.h"
+#include "qt/pivx/qtutils.h"
+#include "clientversion.h"
+
+NavMenuWidget::NavMenuWidget(PIVXGUI *mainWindow, QWidget *parent) :
+    QWidget(parent),
+    ui(new Ui::NavMenuWidget),
+    window(mainWindow)
+{
+    ui->setupUi(this);
+    this->setFixedWidth(100);
+    setCssProperty(ui->navContainer_2, "container-nav");
+    setCssProperty(ui->imgLogo, "img-nav-logo");
+
+    // App version
+    ui->labelVersion->setText("v 4.0.0");//QString(tr("v %1")).arg(QString::fromStdString(FormatFullVersion())));
```

Comment:
Reminder: change to using calculated version instead of static text

Reply:
done